application/views/admin/survey/importSurvey_view.php

```diff
@@ -16,7 +16,16 @@
 
         <!-- error message -->
         <p>
-            <?php echo $sErrorMessage; ?>
+		<?php
+ -        	if(is_array($sErrorMessage))
```

Comment:
Didn't see this .... syntax error
Fixed in https://github.com/LimeSurvey/LimeSurvey/commit/44d3a784efd920d834fe80103ffe990e0801eff4